### PR TITLE
Chore/no ticket]landing page updates

### PR DESCRIPTION
**Pull request by stef-cobaltio**
Updating the copy at the top of the home page and removing the CTA that takes our customers away from the docs site.

## Preview This Change

To see how this change looks in production, scroll down to **Deploy Preview**. Select the link that looks like `https://deploy-preview-<num>--cobalt-docs.netlify.app/`

## Variables

Help us support a “Write once, publish everywhere” single source of truth. If you see a line that looks like:

`{{% asset-categories %}}`

You’ve found a [shortcode](https://gohugo.io/content-management/shortcodes/) that we include in multiple documents.

You’ll find the content of the shortcode in the following directory:

https://github.com/cobalthq/cobalt-product-public-docs/tree/main/layouts/shortcodes

That shortcode has the same base name as what you see in the PR, such as `asset-categories.html`.

## Checklist for PR Author

[ ] Did you check for broken links and alt text?

Be sure to check for broken links and Alt Text issues. We have a partially automated process,
as described in this section of our repository README:
[Test Links and Alt Attributes](https://github.com/cobalthq/cobalt-product-public-docs/blob/main/README.md#test-links-and-alt-attributes).

### CODE REVIEW

**Comment by stef-cobaltio** [comment]
@mtsuji-cobalt this one is ready for review whenever you have time.

**Comment by mtsuji-cobalt** [comment]
Something strange is happening with the footer, I can also take a look into fixing it:
Screenshot 2024-09-05 at 08 52 08

File: content/en/_index.md [hunk 1]
```diff
@@ -19,8 +19,9 @@ cascade:
 
 <div class="jumbotron px-5" style="background-color: #F8F9FB; margin-bottom: 4rem;">
        <h1>Welcome to Cobalt Documentation</h1>
-       <p class="my-3">Deep dive into our documentation to explore Cobalt—your all-in-one platform for pentesting your software.</p>
-       <a class="btn btn-outline-primary rounded" href="https://www.cobalt.io/" target="_blank">Explore Cobalt »</a>
+       <h3>Deep Dive into Cobalt</h3>
```
Comment (mtsuji-cobalt) [inline content/en/_index.md:22]: I think we might not need this extra title line of Deep Dive into Cobalt. The description below already does the job.

File: content/en/_index.md [hunk 2]
```diff
@@ -92,8 +93,13 @@ cascade:
...
-  <h3>Need More Help? Get in Touch!</h3>
-  <p>If you need help, don't hesitate to contact us.</p><ul><li>If you have a named Customer Success Manager, get in touch with them.</li><li>Otherwise, <a href="https://cobaltio.zendesk.com/hc/en-us/requests/new" target="_blank">submit a ticket</a> or send an email to <a href="mailto:support@cobalt.io" target="_blank">support@cobalt.io</a>.</ul>
+{{< alert title="Need Further Assistance? Contact Us!" color="primary" >}}
+If you need help, don't hesitate to reach out directly to your customer success manager, <a href="https://cobaltio.zendesk.com/hc/en-us/requests/new" target="_blank">submit a ticket</a>, or send us an email at <a href="mailto:support@cobalt.io" target="_blank">support@cobalt.io</a>.
+{{< /alert >}}
+</div>
+
+</div>
+
 </div>
```
Comment (mtsuji-cobalt) [inline content/en/_index.md:103]: I *think* you need to delete the these two extra </div> here

**Pull request closed without merging**